### DISCUSSION TITLE
Stuck on server version 32.12

### DISCUSSION THREAD

**Discussion by R3DD3VIL1982** [Ch4r0ne/ARK-Ascended-Server-Manager#40] [General]
Hi im a little stuck, after the winter wonderland update i could no longer find my server in the server list i host the server on my pc but play from my xbox with a friend.
I think the problem is the xboxs have updated to a newer version their on 33.3 wile the server is 32.12 any help would be great

**Comment by R3DD3VIL1982** [comment]
Yeeeeeesssssss iv fixed it i hit install on the panel then update and it worked.
I am sure it tried that before

**Comment by Ch4r0ne** [comment]
Yes, with the latest version is then also fixed, there was a bug in the earlier version.
